config/routes.rb

```diff
@@ -169,6 +169,7 @@
 
     resource :option_set_imports, path: "option-set-imports", only: [:new, :create] do
       collection do
+        post :upload
         get "option-sets-template", as: "template", action: "template", defaults: { format: "xlsx" }
```

Comment:
This is a weird old way to define these routes. Could you please refactor to just `get :template` and quickly manually test the user batch case? I think that's all we need. There is no need for a default format.